server/src/main/java/org/opensearch/common/Rounding.java

```diff
@@ -435,7 +438,9 @@ protected Prepared maybeUseArray(long minUtcMillis, long maxUtcMillis, int max)
                 values = ArrayUtil.grow(values, i + 1);
                 values[i++] = rounded;
             }
-            return new ArrayRounding(values, i, this);
```

Comment:
Add a link to the results so folks have a reference?

Reply:
Done!